cmd/device-hub-cli/cmd_create.go

```diff
@@ -49,6 +49,13 @@ var createCommand = &cobra.Command{
 				params, err = register.DescribeListener(v.Kind)
 			case "endpoint":
 				params, err = register.DescribeEndpoint(v.Kind)
+			case "process":
+				request := proto.StartRequest{
+					Endpoints: []string{},
+					Tags:      map[string]string{},
+				}
+				tags := []string{}
+				return startCall(args, request, tags, cli, in, out)
```

Comment:
I'm really not keen on this case statement - lines 52 to 58 : 
- the other two cases are for validation whilst this specializes for the end of the function e.g. does almost the same bits of code just different.